src/Run/Interop.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.DotNet.Execute
+{
+    internal class Interop
+    {
+        public static bool GetUnixVersion(out string result)
+        {
+            result = null;
+            const string OSReleaseFileName = @"/etc/os-release";
+            if (File.Exists(OSReleaseFileName))
+            {
+                string content = File.ReadAllText(OSReleaseFileName);
+                int idIndex = content.IndexOf("ID");
```

Comment:
Might be nice if you change these to be constants, append an = to the strings and then use `<constant_name>.Length` instead of hard-coding the lengths...